.github/workflows/canary-integration-suite.yml

```diff
@@ -24,6 +24,11 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
   cancel-in-progress: true
 
+  permissions:
+    contents: read
+
+
+
```

Comment:
Let's keep these files following the pattern of one empty line between sections. I see between 0 and 3 across all files.

Reply:
@BlaineEXE  wrote:

> Let's keep these files following the pattern of one empty line between sections. I see between 0 and 3 across all files.

Good point. I tried to find and fix all cases.